src/sql/src/plan/query.rs

```diff
@@ -550,6 +551,128 @@ pub fn plan_insert_query(
     Ok((table.id(), expr.map(map_exprs).project(project_key)))
 }
 
+pub fn plan_copy_from(
+    scx: &StatementContext,
+    table_name: UnresolvedObjectName,
+    columns: Vec<Ident>,
+) -> Result<(GlobalId, RelationDesc, Vec<usize>), anyhow::Error> {
```

Comment:
This would perhaps benefit from being a CoordError so we can return more specific error codes. But that could be improved in a followup.